src/js/azdata/azdataQueryPlan.js

```diff
@@ -18,12 +18,12 @@ azdataQueryPlan.prototype.init = function (container, iconPaths) {
 
     var graph = new azdataGraph(container);
     this.graph = graph;
+    this.rubberband = new mxRubberband(graph);
     graph.centerZoom = false;
+    this.enablePanning(false);
```

Comment:
Why are disabling Panning? From what I know this feature is present in SSMS.

Reply:
I added a hook to enable and disable, but I needed to initially set it as disabled for rectangular zooming to work since that requires clicking and dragging like panning does. I'll set it to enabled by default.